### PR TITLE
Fixed #31428 -- Allowed empty message in management command self.stdout/err proxies.

### PR DESCRIPTION
**Pull request by francoisfreitag**

### CODE REVIEW

File: tests/user_commands/tests.py [hunk 1]
```diff
@@ -42,6 +42,11 @@ def test_command_style(self):
         management.call_command('dance', '--style', 'Jive', stdout=out)
         self.assertIn("I don't feel like dancing Jive.\n", out.getvalue())
 
+    def test_command_outputs_newline(self):
+        out = StringIO()
+        management.call_command('dance', "--newline", stdout=out, verbosity=0)
+        self.assertEqual("\n", out.getvalue())
```
Comment (felixxm) [inline tests/user_commands/tests.py:48]: I don't think that we need a separate test for this PR. It's already tested implicitly for builtin commands, so I've added this to an existing test.

**Comment by felixxm** [comment]
@francoisfreitag Thanks :+1:

**Comment by felixxm** [comment]
Docs failures are not related with this PR, see #12670.

File: tests/user_commands/tests.py (hunk 1 above)
Comment (francoisfreitag) [reply tests/user_commands/tests.py]: Thank you!

**Pull request merged**